jstest/runnable.jobs

```diff
@@ -0,0 +1,22 @@
+ [
+  {
+    "id": "profiles/minimal-profile-build",
+    "patches": false,
+    "coverage": false,
+    "no_memstat": true,
+    "no_flash": true,
+    "no_test": true
+  },
+  {
+    "id": "profiles/target-profile-build",
+    "patches": false,
+    "coverage": false,
+    "no_memstat": true,
+    "no_flash": true,
+    "no_test": true
+  },
+  {
+    "id": "test-build",
+    "patches": true
```

Comment:
You are right. I'd like to create documentation for all the modules that could describe what features are in the modules and how they work. But this could be a separated patch.